File: official/docs/go/tracker/create_tracker.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	tracker, _ := client.CreateTracker(
		&easypost.CreateTrackerOptions{
			TrackingCode: "9400110898825022579493",
			Carrier:      "USPS",
		},
	)

	fmt.Println(tracker)
}

File: official/guides/tracking-guide/java/create-tracker.java
package getting_started;

import java.util.HashMap;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;

public class CreateTracker {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        // Creating a Tracker
        Map<String, Object> params = new HashMap<String, Object>();
        params.put("tracking_code", "EZ4000000004");
        params.put("carrier", "UPS");
        Tracker tracker = Tracker.create(params);
    }
}
